<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="sinister">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>390</width>
     <height>770</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QLabel" name="sin_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sinister</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperature_sin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lämpötila:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="dexter">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>390</width>
     <height>770</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="dex_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dexter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temperature_dex">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="temp_label_dex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lämpötila:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="meas_block">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>50</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QGraphicsView" name="m_graph">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[mittaus]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="p_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P-arvo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="p_value">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_header">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;ra_apb&quot;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
